app/client/stylesheets/_home.scss

```diff
@@ -58,15 +58,15 @@
 		position: absolute;
 	}
 
-	#hero-content {
+	.hero-content {
 		height: 100%;
 		position: relative;
 		color: white;
 		z-index: 3;
-	}
 
-	#hero-content>div {
-		margin-top: -50px;
+		> div {
```

Comment:
don't need ">", the scss compiler will automatically add the ">" because of the nesting